Author: Roland Rosenfeld <roland@debian.org>
Description: Change several paths to fit Debian needs.
 Modify some settings for the Debian package.

--- a/fig2dev/Imakefile
+++ b/fig2dev/Imakefile
@@ -34,7 +34,7 @@
 XCOMM You should point XFIGLIBDIR to the same directory you did when you compiled
 XCOMM and installed xfig.
 
-XFIGLIBDIR =	/usr/local/lib/X11/xfig
+XFIGLIBDIR =	/usr/share/xfig
 
 XCOMM ******
 XCOMM If your system doesn't have the strerror() function (has sys_errlist) then
@@ -53,7 +53,7 @@
 XCOMM inline functions. With the "INLINE" keyword, you should notice that
 XCOMM the display will be a bit faster in complex figures
 
-XCOMM USEINLINE = -DUSE_INLINE
+USEINLINE = -DUSE_INLINE
 
 XCOMM ****************
 XCOMM Change RGB if necessary, to point to your rgb.txt color database
@@ -62,7 +62,7 @@
 XCOMM the full name (rgb.txt) now.
 XCOMM
 
-RGB = $(LIBDIR)/rgb.txt
+RGB = /etc/X11/rgb.txt
 
 XCOMM Comment out the following #define USEPNG if you don't want PNG support
 XCOMM Uncomment the #define for USEPNG if you want to be able to import
@@ -97,7 +97,7 @@
 
 XCOMM ****************
 XCOMM Uncomment the following to set default paper size to A4
-XCOMM DDA4 = -DA4
+DDA4 = -DA4
 
 XCOMM ****************
 XCOMM Uncomment the following to set IBM Graphics Enhancement Cartridge
@@ -141,7 +141,7 @@
 
 #ifdef I18N
 I18N_DEFS = -DI18N 
-FIG2DEV_LIBDIR = /usr/local/lib/fig2dev
+FIG2DEV_LIBDIR = /usr/share/xfig/fig2dev
 I18N_DEV_DEFS = $(I18N_DEFS) -DFIG2DEV_LIBDIR=$(FIG2DEV_LIBDIR) -DFIG2DEV_LIBDIR_STR=\\\"$(FIG2DEV_LIBDIR)\\\"
 
 #endif /* I18N */
@@ -229,7 +229,7 @@
 	else (set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/bitmaps ); fi ; \
 	echo Copying bitmap files for tk to $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
 	for f in * ; do \
-	      $(INSTALL) -c $$f $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
+	      $(INSTALL) -m644 $$f $(DESTDIR)$(XFIGLIBDIR)/bitmaps ; \
 	done) ; 
 
 install.all::
--- a/transfig/Imakefile
+++ b/transfig/Imakefile
@@ -32,7 +32,7 @@
 XCOMM \\usepackage{} command for LaTeX2e. 
 XCOMM The default is to use \\documentstyle{} for LaTeX209.
 
-XCOMM USELATEX2E = -DLATEX2E
+USELATEX2E = -DLATEX2E
 
 XCOMM ******* DON'T CHANGE ANYTHING BELOW THIS POINT *******
 
